src/blockdata/transaction.rs

```diff
@@ -538,7 +537,8 @@ impl Transaction {
         let mut input_weight = 0;
         let mut inputs_with_witnesses = 0;
         for input in &self.input {
-            input_weight += scale_factor*(32 + 4 + 4 + // outpoint (32+4) + nSequence
+            input_weight += scale_factor
+                * (32 + 4 + 4 + // outpoint (32+4) + nSequence
                 VarInt(input.script_sig.len() as u64).len() +
                 input.script_sig.len());
```

Comment:
Here curiously rustfmt is mixing "end lines with operators" and "start lines with operators", presumably because it is confused by the comment.

Reply:
Manually changing to
```rust
            input_size += 32 + 4 + 4 // outpoint (32+4) + nSequence
                + VarInt(input.script_sig.len() as u64).len()
                + input.script_sig.len();
```

Reply:
I fixed this one manually, and more of the same in this file, added a separate patch for quick review. I'm going to convert this to draft while I try and make this PR more reviewable.